pkgs/by-name/sq/sql-migrate/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sql-migrate";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "rubenv";
+    repo = "sql-migrate";
+    rev = "v${version}";
+    hash = "sha256-l9HW8L9khr+UfxlhH9z2mrzjemlqRbnnM7rPsf9SCtg=";
+  };
+
+  sourceRoot = "${src.name}";
```

Comment:
This shouldn’t be needed in this case, [as per the manual](https://nixos.org/manual/nixpkgs/stable/#ssec-unpack-phase):

> if neither of sourceRoot and setSourceRoot are set, unpackPhase of the generic builder checks that the unpacking produced a single directory and moves the current working directory into it.

```suggestion
```